hermes.py

```diff
@@ -211,8 +211,17 @@ def extract_bucket_key_from_s3_url(url):
 
 def decrypt_and_write_to_file(encrypted_filename, decrypted_output_filename, secret_key_file):
     decrypted_dict = None
+    secret_keys = secret_key_file.split(',')
     with open(encrypted_filename, "r") as f:
-        decrypted_dict = asym_crypto_yaml.load(f, secret_key_file)
+        for secret_key in secret_keys:
+            try:
+                decrypted_dict = asym_crypto_yaml.load(f, secret_key)
```

Comment:
secret_key may be None in some cases.

asym_crypto_yaml.load(f, None) should still run in these cases, as asym-crypto-yaml will basically function as a yaml.load when no key is provided.  Hermes should also function when no key is provided, so we still want to call asym_crpyo_yaml.load even when the key is None and write it to a file, it will essentially just pass the yaml through without decrypting when no keys are provided.